packages/tasks/src/local-apps.ts

```diff
@@ -143,6 +144,15 @@ const snippetNodeLlamaCppCli = (model: ModelData, filepath?: string): LocalAppSn
 	];
 };
 
+const snippetOllama = (model: ModelData, filepath?: string): string => {
+	if (filepath) {
+		const quantLabel = parseGGUFQuantLabel(filepath);
+		const ollamatag = quantLabel ? `:${quantLabel}` : "";
+		return `ollama run hf.co/${model.id}${ollamatag}`;
+	}
+	return `ollama run hf.co/${model.id}{{OLLAMA_TAG}}`;
```

Comment:
maybe document it line 55 (where you document `GGUF_FILE`) (can be in a later PR if you prefer)

Reply:
handled in c2691b00d212a2f5b811ab9136fc040724d63043